bindings/azure/signalr/signalr.go

```diff
@@ -0,0 +1,195 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package signalr
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/gbrlsnchs/jwt/v3"
```

Comment:
Is this the defactor JWT package for golang projects?

Reply:
If there'a a "defacto" package for JWT for Go I'd say its [this](https://github.com/dgrijalva/jwt-go) one..

@fbeltrao would it be possible to make the switch unless there are special reasons to use the current one?

Reply:
Wasn’t sure about that one too. Thanks for the tip.